[IDP-1335]: Added the IDP section under supported-platforms-and-technologies page (#3649)
harness/developer-hub: docs/get-started/supported-platforms-and-technologies.md

  - Harness provides [token key rotation natively](/docs/platform/automation/api/add-and-manage-api-keys#rotate-tokens).
- Continuous Verification: TUF recommends the verification of deployments akin to [Harness Continuous Verification](/docs/continuous-delivery/verify/verify-deployments-with-the-verify-step).

## Internal Developer Portal

Harness Internal Developer Portal integrates with a number of third-party providers to provide a single pane of glass for developers.

### Plugins for IDP

Harness IDP supports a number of plugins to integrate the software catalog with third-party providers. Please find the growing list of [supported plugins](https://developer.harness.io/docs/category/available-plugins). This is a subset of the [Backstage plugin marketplace](https://backstage.io/plugins).

### Git providers that IDP supports

Any software component can be registered in the catalog by using a YAML file stored in the following Git providers:

- GitHub
- GitLab
- Bitbucket
- Azure Repos

### CI/CD providers that IDP supports

Harness IDP is agnostic to your CI/CD provider. Following are some of the examples. 

> - Gitlab

Reviewer: ```suggestion
- GitLab
```

- CircleCI
- GitHub Actions 

:::info

> We suggest to use Harness CI/CD for a better out-fo-the-box experience. 

Reviewer: ```suggestion
Harness CI and CD has a seamless out-of-the-box experience with IDP.
```
